cpclib/cfwi/local.Makefile

```diff
@@ -7,7 +7,7 @@ all_fw_calls_official_list.csv: s968se15.pdf
 	pdfgrep '^[0-9][0-9]*:' "$<" | sed 's/^\([0-9]*\): \(.*[^ ]\) *#\(.*\)$$/\1,\2,\3/' >$@.tmp && mv -vf $@.tmp $@
 
 s968se15.pdf:
-	wget -S http://cpctech.cpc-live.com/s968se15.pdf
+	wget -S http://cpctech.cpcwiki.de/s968se15.pdf
```

Comment:
Thanks for the PR. URL still nonfunctional. :-/

```bash
make -f local.Makefile s968se15.pdf 

wget -S http://cpctech.cpcwiki.de/s968se15.pdf
--2020-09-08 11:28:11--  http://cpctech.cpcwiki.de/s968se15.pdf
Resolving cpctech.cpcwiki.de (cpctech.cpcwiki.de)... 213.202.247.37
Connecting to cpctech.cpcwiki.de (cpctech.cpcwiki.de)|213.202.247.37|:80... connected.
HTTP request sent, awaiting response... 
  HTTP/1.1 404 Not Found
  Date: Tue, 08 Sep 2020 09:28:13 GMT
  Server: Apache
  Last-Modified: Thu, 06 Oct 2016 19:12:44 GMT
  ETag: "59a-53e37140be300"
  Accept-Ranges: bytes
  Content-Length: 1434
  X-Powered-By: PleskLin
  Keep-Alive: timeout=5, max=100
  Connection: Keep-Alive
  Content-Type: text/html
2020-09-08 11:28:12 ERROR 404: Not Found.

make: *** [local.Makefile:10: s968se15.pdf] Error 8
```

Reply:
Haha, whoops, didn't actually explicitly check that one. Mea culpa.